src/pages/about-page/index.tsx

```
@@ -0,0 +1,65 @@
+        real-time interfaces, data-driven dashboards, and structured business
+        workflows across trading and enterprise systems. Strong in state
+        management, API integration, and clean, maintainable architecture.
+        Committed to delivering reliable, production-ready solutions in Agile
+        environments.
+      </div>
+      <section className="mb-4">
+        <h1 className="text-3xl section mb-2">Certificate</h1>
+        <div className="grid grid-cols-3 gap-2">
+          {certificates.map((cer) => {
+            return cer.name === "invalid" ? (
+              <div className="flex p-5 border flex-col justify-center items-center" />
```

Biome (2.3.14)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)